src/openapi/streaming/subscription.js

```diff
@@ -55,7 +71,11 @@ function subscribe() {
 
     log.debug(LOG_AREA, 'starting..', { serviceGroup: this.serviceGroup, url: this.url });
     this.currentState = STATE_SUBSCRIBE_REQUESTED;
-    this.transport.post(this.serviceGroup, this.url, null, { body: data })
+
+    const urlTemplate = getUrlTemplate(this.url, this.subscriptionData);
+    const templateArgs = getTemplateArgs(this.subscriptionData);
+
+    this.transport.post(this.serviceGroup, urlTemplate, templateArgs, { body: data })

```

Comment:
the url template and template args are really only there as helpers and to to allow clearer code..

In this case, I'm not sure we need it.. shall we just have a getUrl method?

Reply:
I'll need to check it. I assumed they also normalize,escape url.